cypress/e2e/pages/board/board.helper.js

```diff
@@ -63,8 +77,9 @@ export class BoardHelper {
 
     static createBoard(board_name) {
         this.clickCreateBoardButton();
+        this.clickHeaderCreateBoardButton();
         this.insertBoardTitle(board_name);
-        this.clickSubmitBoardButton();
+        this.clickSubmitBoardButton(board_name);
```

Comment:
```suggestion
        this.clickSubmitBoardButton();
```
No necesitar aumentar esto ya esta usando el metodo y solo es de accion  no requiere name inicialmente